src/components/CurriculumComponents/CurriculumModalCloseButton/CurriculumModalCloseButton.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { FC } from "react";
+
+import Box from "@/components/SharedComponents/Box";
+import IconButton from "@/components/SharedComponents/Button/IconButton";
+
+type CurriculumModalCloseButtonProps = {
+  onClose: () => void;
+};
+
+export const CurriculumModalCloseButton: FC<
+  CurriculumModalCloseButtonProps
+> = ({ onClose }) => {
+  return (
+    <Box $position={"absolute"} $top={[16, 32]} $right={[16, 32]}>
```

Comment:
I don't think this should live in the component. Generally you want positioning to stay outside the component as components should be able to live in other places.

So put the `<Box/>` into the two places in `./src/components/SharedComponents/SubjectPhasePicker/SubjectPhasePicker.tsx`